src/legacy/core_plugins/data/public/search/fetch/handle_response.test.ts

```diff
@@ -18,9 +18,9 @@
  */
 
 import { handleResponse } from './handle_response';
-import { toastNotifications } from '../../notify/toasts';
+import { toastNotifications } from 'ui/notify/toasts';
```

Comment:
Ah, just realized this is another thing that is going to need to be replaced by the `core` service.

Reply:
Yeah that's why I was updating these to say `ui/` -- so that I wouldn't forget :)